src/amo/components/SearchForm/index.js

```diff
@@ -247,7 +247,8 @@ export class SearchFormBase extends React.Component {
         })}
         method="GET"
         onSubmit={this.handleSearch}
-        ref={(ref) => { this.form = ref; }}
+        ref={(ref) => { this.form = ref; }
+        data-no-csrf}
```

Comment:
It looks like there is an issue here. It should be:

```
ref={(ref) => { this.form = ref; }}
data-no-csrf
```

Having a unit test for it would be nice too (an assertion in an existing test case would do the trick)

Reply:
Ha, was wondering about that. Will fix that.
So I've not written / updated any tests on addons before - which one would be a good one to base mine on?

Reply:
Syntax fixed